src/scripts/main.js

```diff
@@ -1 +1,86 @@
 'use strict';
+
+let isPromise1Clicked = false;
+
+// eslint-disable-next-line prefer-const
+let isLeftMbClicked = false;
+// eslint-disable-next-line prefer-const
+let isRightMbClicked = false;
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    isPromise1Clicked = true;
+    resolve();
+  });
+
+  if (isPromise1Clicked === false) {
+    setTimeout(reject, 3000);
+  }
+});
+
+promise1.then(() => {
+  const div = document.createElement('div');
+
+  div.innerHTML = 'First promise was resolved';
+  div.className = 'success';
+  div.setAttribute('data-qa', 'notification');
+
+  document.body.append(div);
+});
+
+promise1.catch(() => {
+  const div = document.createElement('div');
+
+  div.innerHTML = 'First promise was rejected';
+  div.className = 'error';
+  div.setAttribute('data-qa', 'notification');
+
+  document.body.append(div);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  document.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+    resolve();
+  });
+
+  document.addEventListener('click', (e) => {
+    resolve();
+  });
```

Comment:
The `promise2` resolves on both right-click and left-click events. If the intention is to resolve only on a specific event, you should adjust the logic accordingly.